### PR TITLE
remove annotations

### PR DESCRIPTION
**Pull request by Youngestdev**

### CODE REVIEW

File: server/organisation.go [hunk 1]
```diff
@@ -52,19 +41,6 @@ func (a *ApplicationHandler) GetOrganisation(w http.ResponseWriter, r *http.Requ
 // @Router /api/v1/organisations [get]
 func _() {}
```
Comment (Subomi) [inline server/organisation.go:42]: @Youngestdev You can take this one out too.

Comment (Subomi) [inline server/organisation.go:42]: @Youngestdev, You can take this one out too.

**Review by Subomi — CHANGES_REQUESTED** [review]

File: server/organisation.go (hunk 1 above)
Comment (jirevwe) [reply server/organisation.go:42]: @danvixent isn't this the one you added, I can't remember why again

Comment (danvixent) [reply server/organisation.go:29]: @Subomi @Youngestdev please we shouldn’t remove those.

**Pull request closed without merging**